content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -240,6 +240,24 @@ Now it's time to take a look at the metrics we're working with.
 _If you don't have a number of training epochs defined and you don't terminate
 the process, the experiment will run for 100 epochs._
 
+### Caching checkpoints
+
+We can automatically push the checkpoints' code & data to your Git & DVC remotes
```

Comment:
We don't use & much, do we? I also would prefer it from time to time.

Reply:
I avoid it. However in this case having 2 *and*s close together looks confusing.

Reply:
We don't but that's stylistic so up to @casperdcl since he's assigned.